<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractWizard</class>
 <widget class="QWizard" name="ExtractWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер извлечения ключа</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HaveHelpButton|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="page2">
   <property name="title">
    <string>Биометрическая аутентификация</string>
   </property>
   <property name="subTitle">
    <string>Введите биометрический образ для продолжения</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="_icn2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources.qrc">:/res/48px-Preferences-desktop-peripherals.svg.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="_btnAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/res/list-add.png</normaloff>:/res/list-add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="_btnClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/res/edit-clear.png</normaloff>:/res/edit-clear.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="8">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
    </item>
    <item row="0" column="5" colspan="4">
     <widget class="QTableWidget" name="_tblScheme">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>72</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Новая строка</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Новый столбец</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Новый столбец</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Новый столбец</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Новый столбец</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Новый столбец</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page3">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>ExtractWizard</receiver>
   <slot>addImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractWizard</sender>
   <signal>currentIdChanged(int)</signal>
   <receiver>ExtractWizard</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_btnClear</sender>
   <signal>clicked()</signal>
   <receiver>ExtractWizard</receiver>
   <slot>clearImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeContainer()</slot>
  <slot>addImage()</slot>
  <slot>clearImage()</slot>
  <slot>changePage()</slot>
 </slots>
</ui>
